src/components/core/ZoomImage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { type ImgHTMLAttributes } from "react";
+
+export interface ZoomImageProps extends ImgHTMLAttributes<HTMLImageElement> {
+  src: string;
+  alt: string;
+  className?: string;
+  imgClassName?: string;
+}
+
+export function ZoomImage({
+  src,
+  alt,
+  className = "",
+  imgClassName = "",
+  ...props
+}: ZoomImageProps) {
+  return (
+    <div className={`w-full overflow-hidden ${className}`}>
+      <img
+        src={src}
+        alt={alt}
+        className={`w-full h-full object-cover object-center transition-transform group-hover:scale-[1.02] hover:scale-[1.02] ${imgClassName}`}
```

Comment:
- I was using `scale-105` for image hover. I'm worried that `[1.02]` won't be enough to see in a lot of cases.
- Also, can you combine all classes with `cx` so that `undefined` doesn't leak into active classNames
- I don't think we should use `group-` unless that group is defined within the same component.
- I think we need `focus-visible:scale-[whatever-we-agree-on]` as well 


```suggestion
        className={cx("w-full h-full object-cover object-center transition-transform hover:scale-105 focus-visible:scale-105", imgClassName)}
```

Same with the `div` above.